src/index.mako.html

```diff
@@ -699,6 +699,7 @@
           resolutions: JSON.parse(${resolutions}),
           defaultEpsg: '${default_epsg}',
           defaultEpsgExtent: JSON.parse(${default_epsg_extend}),
+          defaultElevationModel: '${default_elevation_model}'
```

Comment:
Either remove the quotes here or in the Makefile (`DEFAULT_ELEVATION_MODEL ?= 'COMB'`), otherwise you'll end up with 2 quotes in the html...and most browsers won't swallow that.